Sources/NetworkProtection/KeyManagement/NetworkProtectionTokenStore.swift

```diff
@@ -42,7 +42,7 @@ public protocol NetworkProtectionTokenStore {
 public final class NetworkProtectionKeychainTokenStore: NetworkProtectionTokenStore {
     private let keychainStore: NetworkProtectionKeychainStore
     private let errorEvents: EventMapping<NetworkProtectionError>?
-    private let isSubscriptionEnabled: Bool
+    private let useAccessTokenProvider: Bool
```

Comment:
This is still used by the macOS system extension, so I'm renaming it away from subscription launch-related language.